### PR TITLE
Added macos hack

### PR DESCRIPTION
**Pull request by ArneTR**
Suggestion for making this work on macOS

Currently i hardcoded the user_id 501. This must be made dynamic.

I also did rewrite to urllib to use system packages.

### CODE REVIEW

File: macos/log/carbon-cpu-macos.err [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/bin/bash: /Users/arne/Sites/green-coding/carbon-intensity-cpu/macos/carbon-cpu.sh: No such file or directory
```
Comment (ribalba) [inline macos/log/carbon-cpu-macos.err]: Why is this here?

**Review by ribalba — APPROVED** [review]
LGMT except the log file

**Comment by ArneTR** [comment]
ups. removed
